### DISCUSSION TITLE
Step (10) compare InSAR time series with GNSS

### DISCUSSION THREAD

**Discussion by Abdallahbalushi** [gmtsar/gmtsar#867] [Q&A]
![image](https://github.com/gmtsar/gmtsar/assets/150544398/3bf8640a-562b-4241-9eae-345fd4237748)
When trying to validate InSAR results, I downloaded GPS data as described in step 10 from sopac-csrc.ucad.edu and downloaded _Clean_TrendNeuTimeSeries_comb
for station Yibl. 

![image](https://github.com/gmtsar/gmtsar/assets/150544398/a81b3477-728a-4386-aca1-b9f878f6ceea)

Below is the downloaded GPS station data.. it seems incorrect! 
![image](https://github.com/gmtsar/gmtsar/assets/150544398/4196bf21-401c-4a44-b196-47f5c4edfa2f)

Also, when projecting GNSS time series into LOS, ![image](https://github.com/gmtsar/gmtsar/assets/150544398/aaee5ac1-a5e9-4e4d-b7b7-e55cc5c2b446)

Does this means the exact location of the GPS station or this may be found in the download data file? 

Then, how exactly can the GNSS_velocity.enu file be constructed? 

![image](https://github.com/gmtsar/gmtsar/assets/150544398/30288441-9af0-47a4-9c60-35b697750493)

Thanks in advance.

**Comment by Xiaohua-Eric-Xu** [comment]
The .Z files are likely zip files. You may need to unzip it to see the contents I think.
